src/docs/content/quickstart/helm.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "[Oracle Cloud] Helm"
+date: 2020-03-06T12:27:43-07:00
+draft: false
+weight: 20
+tags:
+  - Kubernetes
+  - OKE
+  - Setup
+  - Helm
+---
+
+This deployment option utilizes [`helm`](https://github.com/helm/helm) for
+configuration and installation onto a [Kubernetes](https://kubernetes.io/)
+cluster. It is _recommended_ to use an
+[Oracle Container Engine for Kubernetes](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)
+cluster, however other **standard** Kubernetes clusters will also work.
+
+```shell--linux-macos
+cd deploy/complete/helm-chart
+```
+
+```shell--win
+dir deploy/complete/helm-chart
+```
+
+> Path for Cloud Native deployment configurations using `helm`
+
+Deploying the complete MuShop application with backing services from Oracle Cloud
+Infrastructure involves the use of the following helm charts:
+
+1. [`setup`](#setup): Installs umbrella chart dependencies on the cluster _(optional)_
+1. **Not supported yet** [`provision`](#provision): Provisions OCI resources integrated with Service Broker _(optional)_
+1. [`mushop`](#deploy-mushop): Deploys the MuShop application runtime
+
+To get started, create a namespace for the application and its associative deployments:
+
+```shell
+kubectl create ns mushop
+```
+
+---
+
+### Setup
+
+{{% content/setup %}}
+
+### Deploy MuShop
+
+Deploying the full application requires cloud backing services from Oracle Cloud Infrastructure.
+These services must be provisioned manually and are configured using kubernetes secrets.
+
+#### Configure
+
+1. Provision an Autonomous Transaction Processing (ATP) database. Once **RUNNING** download the DB Connection Wallet and configure secrets as follows:
+
+   - Create `oadb-admin` secret containing the database administrator password. Used once for schema initializations.
+
+       ```shell
+       kubectl create secret generic oadb-admin \
+         --namespace mushop \
+         --from-literal=oadb_admin_pw='<DB_ADMIN_PASSWORD>'
+       ```
+
+   - Create `oadb-wallet` secret with the Wallet _contents_ using the downloaded `Wallet_*.zip`. The extracted `Wallet_*` directory is specified as the secret contents.
+
+       ```shell
+       kubectl create secret generic oadb-wallet \
+         --namespace mushop \
+         --from-file=<PATH_TO_EXTRACTED_WALLET_FOLDER>
+       ```
```

Comment:
I thought we had automatic wallet handling? Why is this necessary?

Reply:
There are K8s jobs that initializes the database user and assigns a schema to them, like https://github.com/pgressa/oraclecloud-cloudnative/blob/master/deploy/complete/helm-chart/mushop/charts/orders/config/atp.init.sql